CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## v0.4.1
+
+Conduit 0.4.1 builds on the telemetry work from 0.4.0, providing rich,
+Kubernetes-aware observability and debugging.
+
+* Web UI
+  * **New** Automatically-configured Grafana dashboards for Services, Pods,
+    ReplicationControllers, and Conduit Mesh Health. These dashboards also expose
+    new TCP-level stats.
+  * **New** `conduit dashboard` Pod and ReplicationController views.
+* Command-line interface
+  * `conduit tap` has been completely rewritten to accept kubectl-style resources.
+    This command now supports tapping a single Kubernetes resource or group of
+    resources. The `--to` filter can be used to filter tapped requests by destination.
+  * `conduit stat` and `conduit tap` now both support `namespaces`, `deployments`,
+    `replicationcontrollers`, `services`, and `pods`, via kubectl-style resource
+    strings like `deploy`, `deploy/web`, and `deploy web`.
+* Telemetry
+  * **New** Tap support for filtering by and exporting destination metadata.
```

Comment:
Is this section redundant given the CLI and web UI sections above?

Reply:
it is a bit redundant, though it's true that we have new telemetry APIs, and also web and cli features that expose them. /cc @olix0r